Containerfile.bundle

```diff
@@ -28,6 +30,11 @@ LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
 LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v2.0.0+git
 LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 
+LABEL distribution-scope=public
+LABEL release="9.0.4"
```

Comment:
Looks like the `LABEL release="9.0.4"` exist already L#17


https://github.com/openshift/windows-machine-config-operator/pull/2798/commits/9cb4d8afaf2db1654ac958d47628a6356150d465#diff-dbe2414f4999322f05c1ef92e841862599e0fefb5fb5d9f6b2730591a6a1c0a1R17

Reply:
`release` is not the same as version
I added this here https://github.com/openshift/windows-machine-config-operator/commit/0b9b483084fb91a1d2498cc8ea301b103320ef08

Its probably a required label